src/SeleniumLibrary/keywords/element.py

```diff
@@ -164,21 +164,41 @@ def page_should_contain_element(self, locator, message='', loglevel='INFO'):
         self.assert_page_contains(locator, message=message, loglevel=loglevel)
 
     @keyword
+    def get_matching_locator_count(self, locator):
+        """Returns number of elements matching ``locator``
+
+        If you wish to assert the number of matching elements, use
+        `Locator Should Match X Times`. Keyword will always return an integer.
+
+        Examples assumes that locator matches to three elements
+        | ${count} =      | Get Matching Locator Count | name:div_name  |
+        | Should Be True  | ${count} > 2               |                |
+
+        New in SeleniumLibrary 3.0
+        """
```

Comment:
- There's still `.` missing at the end of two sentences.
- The comment before the example isn't that useful anymore. Having just `Example:` would be enough.

Reply:
Done